narwhals/_pandas_like/series.py

```diff
@@ -424,6 +426,23 @@ def std(
         ser = self._native_series
         return ser.std(ddof=ddof)
 
+    def skew(self) -> Any:
```

Comment:
As Marco pointed out in the example, polars and pandas implementation seems to differ. We should try to remap to polars behavior here as well

Reply:
This also has the default Polars behavior (i.e. Biased skewness) now. Is that what you mean by remapping?

Reply:
Maybe this is not explained extensively, but we try to stick to the polars API - which means both signature and behavior matching.

So as mentioned, I am happy to keep the addition of `bias` argument as a follow up, in the meanwhile, everything else should match polars behavior and outcome

Reply:
Got it! The sticking to the Polars API does not seem to be obvious from `CONTRIBUTING.MD`, but that does clarify why we avoid loading libraries such as `numpy`. 

I've adjusted the function to use native operations.